CONTRIBUTING.md

```diff
@@ -32,6 +32,54 @@ We have always been grateful for community contributions that enhance our Steps.
 
 In the meantime, please feel free to share your feature ideas on [Discuss](https://discuss.bitrise.io/c/feature-request).
 
+### Get started
+
+This section helps you get started making changes to an existing Bitrise Step.
+
+Each Step has its own GitHub repository that includes code and the `step.yml` file that defines the configuration of the Step.
+
+Official Bitrise steps are written in [Go](https://golang.org), a simple and efficient programming language. If you are new to the language, we recommend playing with [A Tour of Go](https://tour.golang.org/) and [Go by Example](https://gobyexample.com/).
+
+To get started, let's set up your local environment:
+
+1. [Install Go](https://golang.org/doc/install)
+2. [Install Bitrise CLI](https://devcenter.bitrise.io/bitrise-cli/installation/) - or if it's already installed, make sure it's up to date by running `bitrise update` and `bitrise plugin update`
+3. `git clone` the step repo you want to make changes to
+
+#### Anatomy of a Step
+
+A typical Bitrise step folder structure looks like this:
+
+```
+├── README.md
+├── bitrise.yml       # Workflows for checking code quality and running tests
+├── e2e
+│   └── bitrise.yml   # Workflows that run end-to-end tests on the step
+├── go.mod            # Go module dependencies
+├── go.sum            # Dependency checksums
+├── main.go           # Main entry point of step, execution starts here
+├── main_test.go      # Tests
+├── step.yml          # Metadata describing the step (input, outputs, description)
+├── vendor/           # Vendored dependencies
+```
+
```

Comment:
Can we say this? I think only 3-4 steps have this structure and the step plugin doesn't generate this structure either :( .

Reply:
It's way more than 3-4 now :) 
https://sourcegraph.com/search?q=context:global+repo:bitrise-steplib/.*+repo:contains.file%28e2e/bitrise.yml%29+count:all&patternType=literal

I know this is still not all of our steps in `bitrise-steplib`, but I'd rather write something forward-looking than write about an outdated structure.

And I wouldn't worry about the step generator in this case because this guide is about contributing to existing steps only.